website/versioned_docs/version-2.9/index.md

```diff
@@ -1,27 +1,286 @@
 ---
-sidebar_label: 'Quick start'
-title: 'Quick start'
+sidebar_label: 'Getting started'
+title: 'Getting started'
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 # Apify API client for JavaScript
 
-`apify-client` is the official library to access [Apify API](https://docs.apify.com/api/v2) from your
-JavaScript applications. It runs both in Node.js and browser and provides useful features like
-automatic retries and convenience functions that improve the experience of using the Apify API.
+`apify-client` is the official library to access the [Apify REST API](/api/v2) from your JavaScript/TypeScript applications. It runs both in Node.js and browser and provides useful features like automatic retries and convenience functions that improve the experience of using the Apify API. All requests and responses (including errors) are encoded in JSON format with UTF-8 encoding.
+
+## Pre-requisites
+
+`apify-client` requires Node.js version 16 or higher. Node.js is available for download on the [official website](https://nodejs.org/). Check for your current node version by running:
+
+```bash
+node -v
+```
+
+## Installation
+
+You can install the client via [NPM](https://www.npmjs.com/) or use any other package manager of your choice.
+
+<Tabs groupId="main">
+<TabItem value="npm" label="NPM">
+
+```bash
+npm i apify-client
+```
+
+</TabItem>
+<TabItem value="yarn" label="Yarn">
+
+```bash
+yarn add apify-client
+```
+
+</TabItem>
+<TabItem value="pnpm" label="PNPM">
 
-You can install the client via the [npm package](https://www.npmjs.com/package/apify-client). To do that, simply run `npm i apify-client`.
+```bash
+pnpm add apify-client
+```
+
+</TabItem>
+<TabItem value="bun" label="Bun">
+
+```bash
+bun add apify-client
+```
 
-## Quick Start
+</TabItem>
+</Tabs>
+
+## Authentication and Initialization
+
+To use the client, you need an [API token](/platform/integrations/api#api-token). You can find your token under [Integrations](https://console.apify.com/account/integrations) tab in Apify Console. Copy the token and initialize the client by providing the token (`MY-APIFY-TOKEN`) as a parameter to the `ApifyClient` constructor.
 
 ```js
-const { ApifyClient } = require('apify-client');
+// import Apify client
+import { ApifyClient } from 'apify';
```

Comment:
i would drop the comment, its not adding any value. 

and the import should be from `apify-client`, not the SDK, right? same for all the other imports

Reply:
Haha, nice catch! 😸